### PR TITLE
Update naming from Live to new term Auto Refresh

### PR DESCRIPTION
**Pull request by JV0812**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/search/search-cheat-sheets/log-operators.md [hunk 1]
```diff
@@ -514,7 +514,7 @@ This section provides detailed syntax, rules, and examples for Sumo Logic Opera
    <td><a href="/docs/search/search-query-language/search-operators/sessionize">sessionize</a></td>
    <td>The sessionize operator allows you to use an extracted value from one log message (generated from one system) to find correlating values in log messages from other systems. After you run Sessionize, these related events are displayed on the same page. The thread of logs woven together is called a session.</td>
    <td></td>
-   <td>Not supported in Live Dashboards or any continuous query.</td>
+   <td>Not supported in Auto Refresh Dashboards or any continuous query.</td>
```
Comment (kimsauce) [inline docs/search/search-cheat-sheets/log-operators.md:517]: Should this be lowercase? Consistency with the other instances in this PR.

```suggestion
   <td>Not supported in auto refresh dashboards or any continuous query.</td>
```

File: docs/search/search-cheat-sheets/log-operators.md [hunk 2]
```diff
@@ -104,14 +104,14 @@ Instead, use separate steps:
...
    <td><code>error<br/>| count by _sourceCategory<br/>| fillmissing values("backend", "database", "webapp") in _sourceCategory</code></td>
   </tr>
   <tr>
    <td><a href="/docs/search/search-query-language/group-aggregate-operators/first-last">first and last</a></td>
    <td>First finds the earliest occurrence in search results, and last finds the result that follows all others, based on the sort order for the query.</td>
    <td>_first<br/>_last</td>
-   <td>Not supported in Live Dashboards or any continuous query.</td>
+   <td>Not supported in Auto Refresh Dashboards or any continuous query.</td>
```
Comment (kimsauce) [inline docs/search/search-cheat-sheets/log-operators.md:114]: Should this be lowercase? Consistency with the other instances in this PR.

```suggestion
   <td>Not supported in auto refresh dashboards or any continuous query.</td>
```

File: docs/manage/manage-subscription/cloud-flex-credits-accounts.md [hunk 3]
```diff
@@ -73,7 +73,7 @@ The following table provides a summary list of key features by Sumo Logic Cloud
 
 ## Account Page
 
-The **Account Overview** tab of the  **Account** page for Cloud Flex Credits displays information about your Sumo Logic organization, account type, contract and billing periods, number of credits you've used, as well as live dashboards and real time alerts. The account owner can reassign the Account Owner role from this page.
+The **Account Overview** tab of the  **Account** page for Cloud Flex Credits displays information about your Sumo Logic organization, account type, contract and billing periods, number of credits you've used, as well as auto refresh dashboards and real time alerts. The account owner can reassign the Account Owner role from this page.
```
Comment (kimsauce) [inline docs/manage/manage-subscription/cloud-flex-credits-accounts.md:76]:
```suggestion
The **Account Overview** tab of the **Account** page for Cloud Flex Credits displays information about your Sumo Logic organization, account type, contract and billing periods, number of credits you've used, as well as auto refresh dashboards and real time alerts. The account owner can reassign the Account Owner role from this page.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Looks great! just a couple suggestions.

**Review by kimsauce — APPROVED** [review]

**Review by oshoghi — APPROVED** [review]

**Review by jpipkin1 — APPROVED** [review]
Looks good, @JV0812.

**Pull request merged**